utils/forest_shim/src/address.rs

```diff
@@ -45,6 +45,12 @@ impl Address {
     }
 }
 
+impl quickcheck::Arbitrary for Address {
+    fn arbitrary(g: &mut quickcheck::Gen) -> Self {
+        Address(Address_v3::new_id(u64::arbitrary(g)))
```

Comment:
This is not right. Give some thought to what this function should be doing and ping me in slack if you get stuck.

Reply:
The way I understand it, we want to create an `Address(Address_v3)`. So it seems like my options would be this, `Address::from(Address_v3::new_id(u64::arbitrary(g)))`, or `Address::new_id(u64::arbitrary(g))`.